### ISSUE TITLE
seek the wmt14 en_de data and the bpe codes

### ISSUE THREAD

**Issue by reroze** [facebookresearch/DisCo#12]
it seems that the wmt14_en_de data link in get_data.sh is disabled. The pretrained model also doesn't contain the bpe codes. So I can't get access to the bped data. It will be helpful if updating the data link or providing the  bpe codes.

**Comment by jungokasai** [comment]
Apologies for my late response. You can find bpe codes here: https://github.com/jungokasai/deep-shallow; https://drive.google.com/uc?id=1mNufoynJ9-Zy1kJh2TA_lHm2squji0i9
I used the same data for the two papers.

**Comment by reroze** [comment]
Thanks a lot ! It will be very helpful for my research.

**Comment by reroze** [comment]
Sorry to bother agin. I've downloaded data from link . It is called wmt16.en-de.deep-shallow.dist . So there is one thing I need to make sure, does these data distilled or just origin? That effects the model performance.

**Comment by jungokasai** [comment]
Right, this is distilled data. But I used the same BPE codes as raw data.

**Comment by reroze** [comment]
Thank you! My problem is nearly solved, but I still have a question. The data in paper is wmt14, but the donwloaded file is wmt16, I've seen the same situation in other papers, so are we generally treat wmt14 and wmt16 in de-en dataset the same?

**Comment by jungokasai** [comment]
Good point. So we followed the train/dev./test split from [Vaswani et al., 2017](https://arxiv.org/abs/1706.03762). And indeed, the training data is wmt16, but the test data is newstest2014 (wmt2014). So this is right.
